### DISCUSSION TITLE
Using a Synology NAS drive as a music provider (with HA in container)

### DISCUSSION THREAD

**Discussion by usbrit** [music-assistant/support#1859] [Q&A] [answered]
I have HA running in a container on my Synology NAS. I can't figure out the correct mapping to add a drive on the NAS as a provider.
NAS directory is "NAS_name/data/media/music". 

Do I consider this a local drive as the HA and MA instance is in the container, or do I need to add a network drive? (have tried both with no luck)

Thanks

**Comment by OzGav** [comment]
Sounds like you should be adding a "Filesystem (Remote Share)". Check for permission problems.

**Reply by usbrit** [reply]
OK, checked permissions (OK) and tried every path combination I can think of.
Still getting this error:
![image](https://github.com/music-assistant/hass-music-assistant/assets/53425475/f472e459-f965-47e1-8e8e-53684892933c)
Can someone confirm what the path syntax is for a Synology directory? I'm new to Linux file structure.
This is the folder location on the NAS:
![image](https://github.com/music-assistant/hass-music-assistant/assets/53425475/f6c050b8-eeb0-4cbf-a3e3-6d21de255cb1)
Thanks, sorry for the dumb questions.

**Reply by OzGav** [reply]
Your share entry is incorrect. You have to actually share the directory and then use whatever you call the share in that field (where you currently have data/media)

https://kb.synology.com/en-global/DSM/help/DSM/AdminCenter/file_share_create?version=7

**Reply by usbrit** [reply]
@OzGav Thanks, moving in the right direction i think. The share did exist already, so tried the following and got a new error (after some processing time)
![image](https://github.com/music-assistant/hass-music-assistant/assets/53425475/a9763eb1-d867-412b-833e-04ac4dea8850)
(IP is a tailscale address.
Any further ideas?

**Reply by OzGav** [reply, marked as answer]
I would Google that error. And make sure that IP address is correct. You should be able to connect to that IP with any device on your network so try and connect with your laptop or a phone for example.

**Comment by usbrit** [comment]
> I would Google that error. And make sure that IP address is correct. You should be able to connect to that IP with any device on your network so try and connect with your laptop or a phone for example.

IP address is good. Works on all other devices.

**Reply by OzGav** [reply]
Is MA on the same subnet as the server?
